README.md

```diff
@@ -27,8 +31,8 @@ const printOptions = {
 };
 ```
 
-## Generating Code
-### Node
+### Generating Code
```

Comment:
```suggestion
## Generating RPC Code
```

Also should we move any printing explanation to not be in between the generation of code?